### DISCUSSION TITLE
Err on include format. a?

### DISCUSSION THREAD

**Discussion by rwl7532** [jkotlinski/durexforth#534] [Q&A] [answered]
I try to include format.
I get a?

Some similar includes yield similar.

Am I missing something fundamental.?

**Comment by Whammo** [comment]
It is included by default. I don't know why it errors when it's included again.

**Reply by rwl7532** [reply]
Is there a list of included by default?

**Reply by Whammo** [reply]
In the documentation, under: 'Configuring durexForth'.

**Reply by ekipan** [reply]
Or if you want to know exactly where in the source, check [`forth/base.fs`](https://github.com/jkotlinski/durexforth/blob/master/forth/base.fs), it's the first file loaded as durexForth builds itself. Search for `include` and `included`

**Comment by jkotlinski** [comment, marked as answer]
Hi, the error is because format.fs incorrectly assumes that the `BASE` is `HEX`.

A proper fix would be to change the number `a` to `$a`: c3892887a7181f6df6fbd26dd4d7a1ddcf3317e4

**Reply by rwl7532** [reply]
Made the change, :m!format and include format gave no error.
Thank you
